src/app/components/ReclaimEscrowForm/index.tsx

```diff
@@ -109,8 +110,14 @@ export const ReclaimEscrowForm = memo((props: Props) => {
             }
             reverse
           />
+          <Box align="end" margin={{ top: 'xsmall' }}>
```

Comment:
Can we later add a feature, upon clicking the amount, it would fill in the amount field for you.

Reply:
That would be nice

also, should make this and paratimes "Max" button consistent
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/8ef8747b-5f5e-4017-ae32-c2a8f0338aab)